server/adaptors/integrations/__data__/repository/aws_waf/assets/create_skipping_index-1.0.0.sql

```diff
@@ -0,0 +1,12 @@
+CREATE SKIPPING INDEX ON {table_name} (
+  `timestamp` VALUE_SET, 
```

Comment:
I would remove the time from the index since it may be part of the bucket partition already

Reply:
If we assume it's not part of the partition, minmax seems better than valueset